source/regression1.md

```diff
@@ -294,17 +294,15 @@ of a house that is 2,000 square feet.
 ```
 
 ```{code-cell} ipython3
-nearest_neighbors = (
-    small_sacramento.assign(diff=(2000 - small_sacramento["sqft"]).abs())
-    .nsmallest(5, "diff")
-)
-
-nearest_neighbors
+small_sacramento["dist"] = (2000 - small_sacramento["sqft"]).abs()
+small_sacramento.nsmallest(5, "dist")
 ```
 
 ```{code-cell} ipython3
 :tags: [remove-cell]
 
+nearest_neighbors = small_sacramento.nsmallest(5, "dist")
+
```

Comment:
````suggestion
small_sacramento["dist"] = (2000 - small_sacramento["sqft"]).abs()
nearest_neighbors = small_sacramento.nsmallest(5, "dist")
nearest_neighbors
```

```{code-cell} ipython3
:tags: [remove-cell]


````

We use `nearest_neighbors` in the next visible cell, so we should show how we define it